examples/arkode/CXX_manyvector/ark_sod_lsrk.cpp

```diff
@@ -0,0 +1,568 @@
+    p[i] = udata.eos(w1d[i][0], w1d[i][1], w1d[i][2], w1d[i][3], w1d[i][4]);
+
+  // compute Roe-average state at face:
+  //   wbar = [sqrt(rho), sqrt(rho)*vx, sqrt(rho)*vy, sqrt(rho)*vz, (e+p)/sqrt(rho)]
+  //          [sqrt(rho), mx/sqrt(rho), my/sqrt(rho), mz/sqrt(rho), (e+p)/sqrt(rho)]
+  //   u = wbar_2 / wbar_1
+  //   v = wbar_3 / wbar_1
+  //   w = wbar_4 / wbar_1
+  //   H = wbar_5 / wbar_1
+  rhosqrL   = sqrt(w1d[2][0]);
+  rhosqrR   = sqrt(w1d[3][0]);
+  rhosqrbar = HALF * (rhosqrL + rhosqrR);
+  u         = HALF * (w1d[2][1] / rhosqrL + w1d[3][1] / rhosqrR) / rhosqrbar;
+  v         = HALF * (w1d[2][2] / rhosqrL + w1d[3][2] / rhosqrR) / rhosqrbar;
+  w         = HALF * (w1d[2][3] / rhosqrL + w1d[3][3] / rhosqrR) / rhosqrbar;
+  H = HALF * ((p[2] + w1d[2][4]) / rhosqrL + (p[3] + w1d[3][4]) / rhosqrR) /
+      rhosqrbar;
+
+  // compute eigenvectors at face (note: eigenvectors for tracers are just identity)
+  qsq  = u * u + v * v + w * w;
+  gamm = udata.gamma - ONE;
+  csnd = gamm * (H - HALF * qsq);
+  cinv = ONE / csnd;
+  for (i = 0; i < 5; i++)
+    for (j = 0; j < 5; j++)
+    {
+      RV[i][j] = ZERO;
+      LV[i][j] = ZERO;
+    }
+
+  RV[0][0] = ONE;
+  RV[0][3] = ONE;
+  RV[0][4] = ONE;
+
+  RV[1][0] = u - csnd;
+  RV[1][3] = u;
+  RV[1][4] = u + csnd;
+
+  RV[2][0] = v;
+  RV[2][1] = ONE;
+  RV[2][3] = v;
+  RV[2][4] = v;
+
+  RV[3][0] = w;
+  RV[3][2] = ONE;
+  RV[3][3] = w;
+  RV[3][4] = w;
+
+  RV[4][0] = H - u * csnd;
+  RV[4][1] = v;
+  RV[4][2] = w;
+  RV[4][3] = HALF * qsq;
+  RV[4][4] = H + u * csnd;
+
+  LV[0][0] = HALF * cinv * (u + HALF * gamm * qsq);
+  LV[0][1] = -HALF * cinv * (gamm * u + ONE);
+  LV[0][2] = -HALF * v * gamm * cinv;
+  LV[0][3] = -HALF * w * gamm * cinv;
+  LV[0][4] = HALF * gamm * cinv;
+
+  LV[1][0] = -v;
+  LV[1][2] = ONE;
+
+  LV[2][0] = -w;
+  LV[2][3] = ONE;
+
+  LV[3][0] = -gamm * cinv * (qsq - H);
+  LV[3][1] = u * gamm * cinv;
+  LV[3][2] = v * gamm * cinv;
+  LV[3][3] = w * gamm * cinv;
+  LV[3][4] = -gamm * cinv;
+
+  LV[4][0] = -HALF * cinv * (u - HALF * gamm * qsq);
+  LV[4][1] = -HALF * cinv * (gamm * u - ONE);
+  LV[4][2] = -HALF * v * gamm * cinv;
+  LV[4][3] = -HALF * w * gamm * cinv;
+  LV[4][4] = HALF * gamm * cinv;
+
+  // compute fluxes and max wave speed over stencil
+  alpha = ZERO;
+  for (j = 0; j < STSIZE; j++)
+  {
+    u          = w1d[j][1] / w1d[j][0];  // u = vx = mx/rho
+    flux[j][0] = w1d[j][1];              // f_rho = rho*u = mx
+    flux[j][1] = u * w1d[j][1] + p[j];   // f_mx = rho*u*u + p = mx*u + p
+    flux[j][2] = u * w1d[j][2];          // f_my = rho*v*u = my*u
+    flux[j][3] = u * w1d[j][3];          // f_mz = rho*w*u = mz*u
+    flux[j][4] = u * (w1d[j][4] + p[j]); // f_et = u*(et + p)
+    csnd  = sqrt(udata.gamma * p[j] / w1d[j][0]); // csnd = sqrt(gamma*p/rho)
+    alpha = max(alpha, abs(u) + csnd);
+  }
+
+  // fp(x_{i+1/2}):
+
+  //   compute right-shifted Lax-Friedrichs flux over left portion of patch
+  for (j = 0; j < 5; j++)
```

Comment:
Add braces `{}` around the for loop bodies (see Rule 13 https://sundials.readthedocs.io/en/latest/developers/style_guide/SourceCode.html#coding-conventions-and-rules).